===================================================================
RCS file: /cvsroot/guifications/guifications2/src/gf_notification.c,v
retrieving revision 1.45
retrieving revision 1.46
diff -u -r1.45 -r1.46
--- guifications/guifications2/src/gf_notification.c	2005/01/01 01:06:38	1.45
+++ guifications/guifications2/src/gf_notification.c	2005/01/07 20:42:56	1.46
@@ -163,7 +163,7 @@
 	parent = xmlnode_new("notification");
 	xmlnode_set_attrib(parent, "type", notification->n_type);
 
-	xmlnode_set_attrib(parent, "type", (notification->use_gtk) ? "1" : "0");
+	xmlnode_set_attrib(parent, "use_gtk", (notification->use_gtk) ? "1" : "0");
 
 	if(notification->background)
 		xmlnode_set_attrib(parent, "background", notification->background);
